crates/api/src/externals.rs

```diff
@@ -191,27 +190,44 @@ impl Global {
         let (wasmtime_handle, wasmtime_export) = generate_global_export(store, &ty, val)?;
         Ok(Global {
             store: store.clone(),
-            ty,
             wasmtime_export,
             wasmtime_handle,
         })
     }
 
     /// Returns the underlying type of this `global`.
-    pub fn ty(&self) -> &GlobalType {
-        &self.ty
+    pub fn ty(&self) -> GlobalType {
```

Comment:
Out of curiosity, how come this changed? Is the intention to avoid storing the `GlobalType` next to the `ExportGlobal` since they're basically inferred from one another?

Reply:
This is because `Module::exports` is now giving you results by value instead of by reference, and without this a lot of common scenarios have to do awkward things to avoid taking a reference of a temporary.